packages/react-interactions/accessibility/src/FocusTable.js

```diff
@@ -162,10 +186,15 @@ export function createFocusTable(scope: ReactScope): Array<React.Component> {
               if (rows !== null) {
                 if (rowIndex > 0) {
                   const row = rows[rowIndex - 1];
-                  focusCellByIndex(row, cellIndex);
-                  event.preventDefault();
+                  focusCellByIndex(row, cellIndex, event);
                 } else if (rowIndex === 0) {
-                  triggerNavigateOut(currentCell, 'up', event);
+                  const wrap = getTableWrapProp(currentCell);
+                  if (wrap) {
+                  } else {
```

Comment:
This empty if looks a bit confusing? Is there a mistake?

Reply:
Fixed :)